<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProxyAppWindowClass</class>
 <widget class="QMainWindow" name="ProxyAppWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1161</width>
    <height>857</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ProxyApp</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="responseButton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>610</y>
      <width>121</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Send response</string>
    </property>
   </widget>
   <widget class="QPushButton" name="spiderButton">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>740</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Spider</string>
    </property>
   </widget>
   <widget class="QLabel" name="statusLabel">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>800</y>
      <width>1081</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Status</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>451</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Client request headers</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="spiderUrlText">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>741</y>
      <width>801</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>710</y>
      <width>801</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Website URL</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="clientRequestText">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>461</width>
      <height>531</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="dumpButton">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>740</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Dump</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="hostResponseText">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>150</y>
      <width>461</width>
      <height>531</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>120</y>
      <width>451</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Host response headers</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>800</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Status:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="requestButton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>520</y>
      <width>121</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Send request</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="portText">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Select port</string>
    </property>
   </widget>
   <widget class="QPushButton" name="portButton">
    <property name="geometry">
     <rect>
      <x>332</x>
      <y>60</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Listen</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="proxyAppWindow.qrc"/>
 </resources>
 <connections/>
</ui>
